common-rendering/src/components/LastUpdated.tsx

```diff
@@ -6,19 +6,23 @@ const LastUpdated = ({
 	lastUpdated,
 }: {
 	lastUpdatedDisplay: string;
-	lastUpdated: number;
+	lastUpdated: Date;
 }) => {
 	return (
 		<div
-			css={css`
+		css={css`
 				display: flex;
 				align-items: flex-end;
 				${textSans.xxsmall()};
 				color: ${neutral[46]};
-			`}
+				`}
 		>
-			<time dateTime={new Date(lastUpdated).toISOString()}>
-				{`Updated at ${lastUpdatedDisplay}`}
+			<time
+				className="js-last-updated-time"
+				data-last-updated={lastUpdated.toISOString()}
```

Comment:
Do we need this? Semantically, `datetime` is the value which is what we need to format thing, so we could just read that?

Reply:
Sorry, maybe I've misread your comment - are you suggesting I don't use the `toISOString` function? In this PR the lastUpdated prop is a `Date`, so I thought converting it to a string would be best?

Reply:
Ah, no sorry, I meant that you don't need the `data-last-updated` attribute at all. You already have the `datetime` attribute which is more semantic and has the required value

Reply:
Thanks - will update :)